internal/service/mwaa/environment_test.go

```diff
@@ -964,6 +966,8 @@ resource "aws_s3_object" "startup_script" {
 }
 
 
+
+
```

Comment:
Remove this extra whitespace?

Reply:
Good catch. Very strange where that came from...